frontend/src/components/@common/Button/TextButton.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { styled } from 'styled-components';
+import { BORDER_RADIUS, FONT_SIZE } from '~/styles/common';
+
+interface TextButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
+  type: 'button' | 'submit';
+  text: string;
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+  colorType: 'dark' | 'light';
+  disabled?: boolean;
+}
+
+function TextButton({ text, colorType, disabled = false, ...props }: TextButtonProps) {
```

Comment:
아래의 TextButton.defaultProps를 설정하면

```suggestion
function TextButton({ text, colorType, disabled, ...props }: TextButtonProps) {
```

이렇게만 써도 되지 않나요? (몰라서 여쭙는 부분)

Reply:
맞아요! 이 부분은 위의 코멘트에 남긴대로 방식을 하나 정해 획일화 시키도록 하죠..!